Rubberduck.VBEEditor/Extensions/VbeExtensions.cs

```diff
@@ -53,6 +54,36 @@ public static IHostApplication HostApplication(this VBE vbe)
         {
             if (vbe.ActiveVBProject == null)
             {
+                const int ctl_view_host = 106;
+
+                CommandBarControl host_app_control = vbe.CommandBars.FindControl(MsoControlType.msoControlButton, ctl_view_host);
+
+                if (host_app_control == null)
+                {
+                    return null;
+                }
+                else
+                {
+                    switch (host_app_control.Caption)
+                    {
+                        case "Microsoft Excel":
+                            return new ExcelApp();
+                        case "Microsoft Access":
+                            return new AccessApp();
+                        case "Microsoft Word":
+                            return new WordApp();
+                        case "Microsoft PowerPoint":
+                            return new PowerPointApp();
+                        case "Microsoft Outlook":
+                            return new OutlookApp();
+                        case "Microsoft Publisher":
+                            return new PublisherApp();
+                        case "AutoCAD":
+                            return null; //TODO - Confirm the button caption
```

Comment:
Contains seemed a bit flaky, for (future) things like Visio/Visio Viewer, Word and WordPerfect. And I don't know if AutoCAD has anything like AutoCAD Lite/Pro/Viewer.

I guess `case AutoCAD` *could* return a `new AutoCADApp()`, but it's unlikely that I've guessed the caption correctly. It felt safer to leave a placeholder, that in the worst case, doesn't return an App in the edge-case of there not being any projects present, rather then returning an `AutoCADApp` to something that might be expecting an AutoCAD Viewer.

As of this change, we've currently got 3 ways of determining the host Application, none of which is 100% reliable, all of the time. I think we ultimately need to try and get the host app from the processID of the VBE hWnd, as the preferred method, but this fix addresses the situation where VBE is running without any projects.